app/app.go

```diff
@@ -796,6 +796,19 @@ func (app *App) ModuleAccountAddrs() map[string]bool {
 	return modAccAddrs
 }
 
+// BlockedAddrs returns the set of addresses that are not allowed
+// to send and receive funds
```

Comment:
`to send` NewBaseKeeper comment does not say anything like that, are you sure `send` is forbidden?

Reply:
Yes, it's forbidden in the ibc transfer module. That's why we can't send rewards to a provider chain